lib/ruby_lsp/ruby_lsp_rails/runner_client.rb

```diff
@@ -170,20 +170,23 @@ def make_request(request, params = nil)
         read_response
       end
 
-      sig { params(request: String, params: T.nilable(T::Hash[Symbol, T.untyped])).void }
+      # Messages expect a response back
+      sig { overridable.params(request: String, params: T.nilable(T::Hash[Symbol, T.untyped])).void }
       def send_message(request, params = nil)
         message = { method: request }
         message[:params] = params if params
         json = message.to_json
 
         @stdin.write("Content-Length: #{json.length}\r\n\r\n", json)
       rescue Errno::EPIPE
-        # The server connection died
+        $stderr.puts("Lost connection to the Rails server.")
```

Comment:
If the connection to the server was lost in the middle of the coding session, this will spam the user's output tab with this message and no context of why the server died.

Do we really need it?